FirstProject/css/main.css

```diff
@@ -80,11 +84,66 @@ h2.description_section {
     margin-left: 300px;
 }
 
-/*.row_section {*/
-    /*margin-left:254px;*/
-    /*margin-right: 255px;*/
-/*}*/
-
 fieldset.field_section {
     text-align: center;
-}
\ No newline at end of file
+}
+
+legend {
+    text-align: left;
+    color: #03A9F4;
+}
+
+input[type='radio']{
```

Comment:
As a personal preference, I normally use double quotes in both HTML and CSS. No action item required, whatever works for you is good enough, just as long as you define your own style and stick to it. ;)